### ISSUE TITLE
RTCP: Padding SDES and APP packets to 4-byte length

### ISSUE THREAD

**Issue by jrsnen** [ultravideo/uvgRTP#137]
These should be padded and handled correctly at the receiving end. Currently, this has not been implemented.

**Comment by jrsnen** [comment]
![image](https://user-images.githubusercontent.com/6498638/178420995-f463a0eb-f0df-46d2-866d-df4361f5aaf2.png)

Actually, APP doesn't need padding if this is followed. Only SDES needs the padding, which can be done since items contain their length.

**Comment by wowaser** [comment]
What do you mean when you say that APP packets don't need the padding? So the users have to do their own padding and make sure that it is in 32-bit words?

**Comment by jrsnen** [comment]
@wowaser yes, RFC 3550 mandates that APP data is in multiples of 32-bits. How you accomplish this is up to you. Then you can use the length in the header to get the size of the payload.

**Comment by jrsnen** [comment]
I added padding to SDES, but that was not the only thing missing from SDES. Created a new issue #141. Closing this one.
